### DISCUSSION TITLE
Jellyfin to Panasonic TV (My Home Screen 6.0 Firefox OS)

### DISCUSSION THREAD

**Discussion by ElfsRage** [jellyfin/jellyfin#6945] [General]
Hello,
Could you tell me how I can install jellyfin on a panasonic JZ1500e television (My Home Screen 6.0, Firefox OS)?
Panasonic expert in cinema for many years in colometry in hollywood thanks to its laboratory on site.
https://www.panasonic.com/fr/consumer/televiseurs-savoir/televiseurs/hollywood-to-your-home.html
Thanks!
I love Jellyfin! Very good job!

**Comment by nielsvanvelzen** [comment]
We do not have a dedicated app for this operating system but you should be able to use the built-in browser to use Jellyfin.

**Reply by ElfsRage** [reply]
Good morning,
I seem to have tried but I did not succeed with the embed browser...

**Reply by scootgit** [reply]
Yeah, I can't get rid of the cursor or browser bar on top, too distracting.

**Reply by unix2dossss** [reply]
Likewise, any clue on how you can get rid of the browser bar?

**Reply by dgalli1** [reply]
you can hide the cursor through a bug but it worked for the past 5 years  ¯\_(ツ)_/¯

 Move your cursor to the bottom of the page, till you get the scroll downwards icon.

Then click on the Home button, as soon as it loaded press the back/cancel button.

The cursor should now have disappeared. It will reappear if you press the upward key on your remote.

**Reply by KintsugiUwU** [reply]
Thanks, it’s a nice workaround. Altrough you have to press up and then ok to pause the video as the media buttons on the remote doesn’t seem to work.

Unbelievable that jellyfin isn’t available but plex is. Even more unbelievable that Kodi isn’t available. Kodi runs on a smart fridge. Poor decisions from pansonic

**Reply by dgalli1** [reply]
Yeah, technically the implementation wouldn’t be an issue, but their whole SDK is hidden behind a paywall, which makes it unfeasible for an open-source project to implement a client for it.
